### DISCUSSION TITLE
Suricata 6

### DISCUSSION THREAD

**Discussion by marco3181** [unifi-utilities/unifi-common#160] [Q&A] [answered]
Hello,

I use the UDM Pro with the 1.9.2 firmware version. In this version, Suricata is in version 5.0.5. The version in udm-utilities is a 5.0.3 and the latest version from jasonish/suricata is 6.0.2.
Can we update the current udm-utilities to 6? Is the Suricata 6 compatible with UDM Pro?
By the way, I had to use --force to remove the podman image because of the overlay. And I removed unknown layers manually from json and overlay-images, without any troubles.

Marc

**Comment by boostchicken** [comment, marked as answer]
This is now updated in officall firmware
